### ISSUE TITLE
[BUG] `version` is obsolete warning for two files

### ISSUE THREAD

**Issue by mtrolley** [GhostWriters/DockSTARTer#1748]
# Bug report

**Describe the bug**
Warning that `version` is obsolete appears for two files when running compose.

**To Reproduce**
```
❯ ds -c
2024-03-28 05:38:40 [NOTICE]   Creating environment variables for enabled apps. Please be patient, this can take a while.
2024-03-28 05:38:41 [NOTICE]   Adding compose configurations for enabled apps. Please be patient, this can take a while.
WARN[0000] /home/mark/.docker/compose/.reqs/r1.yml: `version` is obsolete
WARN[0000] /home/mark/.docker/compose/.reqs/r2.yml: `version` is obsolete
```

**Expected behavior**
No warnings.

**Screenshots**

**System**
- OS: Ubuntu
- Version 22.04.4

**Additional context**
N/A

**Comment by spider1163** [comment]
Warnings still happening

**Comment by mtrolley** [comment]
This is still an issue.

**Comment by HalianElf** [comment]
This should be fixed in #1791, if you're still getting it, make sure you're up to date.
